style.css

```diff
@@ -116,14 +116,14 @@ table.dataTable tbody td {
 }
 
 .map-button {
-    font-size: 20px;
+    font-size: 16px;
     margin-left: 15px;
     color: white;
-    padding: 6px 20px;
+    padding: 6px 10px;
     text-align: center;
     text-shadow: 2px 2px 5px black;
-    width: 23.2%;
-    height: 100%;
+    width: 15%;
+    height: 85%;
```

Comment:
Sorry, I changed this to 85%. This makes the button text to be in the middle. At least on my screen. :)